packages/components/src/Select.test.tsx

```diff
@@ -0,0 +1,87 @@
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { renderWithTheme } from './internal/test-utils';
+import { AsyncCreatableSelect, GroupedSelect, Select } from './Select';
+
+describe('Select Component', () => {
+  const options = [
+    { value: 'foo', label: 'Foo' },
+    { value: 'bar', label: 'Bar' },
+  ];
+
+  it('renders', () => {
```

Comment:
these look great, but please add the Tripple-A sections //Arrange, //Act, //Assert to make each test more readable... even in this case here where there is no //Arrange section, still add the act and assert sections, with a blank line in between each section.